test/automation/src/positron/positronPlots.ts

```diff
@@ -4,37 +4,35 @@
 const CURRENT_STATIC_PLOT = '.plot-instance.static-plot-instance img';
 const CLEAR_PLOTS = '.positron-plots-container .positron-action-bar .codicon-clear-all';
-const NEXT_PLOT = '.positron-plots-container .positron-action-bar .codicon-positron-right-arrow';
-const PREVIOUS_PLOT = '.positron-plots-container .positron-action-bar .codicon-positron-left-arrow';
 
 
 /*
  *  Reuseable Positron plots functionality for tests to leverage.
  */
 export class PositronPlots {
 
-	nextPlotButton: PositronBaseElement;
-	previousPlotButton: PositronBaseElement;
-	clearPlotsButton: PositronBaseElement;
-	plotSizeButton: PositronBaseElement;
-	savePlotButton: PositronBaseElement;
-	copyPlotButton: PositronBaseElement;
-	zoomPlotButton: PositronBaseElement;
+	nextPlotButton: Locator;
+	previousPlotButton: Locator;
+	clearPlotsButton: Locator;
+	plotSizeButton: Locator;
+	savePlotButton: Locator;
+	copyPlotButton: Locator;
+	zoomPlotButton: Locator;
 
 	constructor(private code: Code) {
-		this.nextPlotButton = new PositronBaseElement('.positron-plots-container .positron-action-bar .positron-button[aria-label="Show next plot"]', this.code);
-		this.previousPlotButton = new PositronBaseElement('.positron-plots-container .positron-action-bar .positron-button[aria-label="Show previous plot"]', this.code);
-		this.clearPlotsButton = new PositronBaseElement('.positron-plots-container .positron-action-bar .positron-button[aria-label="Clear all plots"]', this.code);
-		this.plotSizeButton = new PositronBaseElement('.positron-plots-container .positron-action-bar .positron-button[aria-label="Auto"]', this.code);
-		this.savePlotButton = new PositronBaseElement('.positron-plots-container .positron-action-bar .positron-button[aria-label="Save plot"]', this.code);
-		this.copyPlotButton = new PositronBaseElement('.positron-plots-container .positron-action-bar .positron-button[aria-label="Copy plot to clipboard"]', this.code);
-		this.zoomPlotButton = new PositronBaseElement('.positron-plots-container .positron-action-bar .positron-button[aria-label="Fill"]', this.code);
+		this.nextPlotButton = this.code.driver.getLocator('.positron-plots-container .positron-action-bar .positron-button[aria-label="Show next plot"]');
+		this.previousPlotButton = this.code.driver.getLocator('.positron-plots-container .positron-action-bar .positron-button[aria-label="Show previous plot"]');
+		this.clearPlotsButton = this.code.driver.getLocator('.positron-plots-container .positron-action-bar .positron-button[aria-label="Clear all plots"]');
+		this.plotSizeButton = this.code.driver.getLocator('.positron-plots-container .positron-action-bar .positron-button[aria-label="Auto"]');
+		this.savePlotButton = this.code.driver.getLocator('.positron-plots-container .positron-action-bar .positron-button[aria-label="Save plot"]');
+		this.copyPlotButton = this.code.driver.getLocator('.positron-plots-container .positron-action-bar .positron-button[aria-label="Copy plot to clipboard"]';
```

Comment:
No idea how this slipped by but there is a missing ) here.  I will just add it in so I can run the tests in CI!

Reply:
We should maybe look at having a commit hook that compiles... seems like something that should be caught automatically.

Reply:
I think I'll have to switch to what you do and manually compile. I find that sometimes the watcher stops re-compiling or misses some files.